Formula/operator-sdk.rb

```diff
@@ -21,23 +21,27 @@ class OperatorSdk < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "b951111c79c494c2e3a4ea06562306d698959a6807b002132c578cca64ad75eb"
   end
 
-  depends_on "go"
+  # TODO(everettraven): update this to be "go" when operator-sdk add support for go 1.18
```

Comment:
This should reference an upstream issue since the TODO won't be seen

Reply:
I changed this in commit https://github.com/Homebrew/homebrew-core/pull/100353/commits/5a4283ab2399d369e2d3b942c946d4aa348e099a

Let me know if it needs any further changes